docs/ros/ros/installation/controller.mdx

```diff
@@ -0,0 +1,76 @@
+---
+title: Joystick Controller Pairing
+sidebar_label: Joystick Controller
+sidebar_position: 3
+toc_min_heading_level: 2
+toc_max_heading_level: 4
+---
+
+Joystick controllers are used for teleoperation; they allow you to remotely drive the Jackal, whether it is a physical Jackal robot,
```

Comment:
Is this Jackal-specific language intentional or should we generalize?

Reply:
Copy paste error. Will fix.